### ISSUE TITLE
Please add changelog into citra 

### ISSUE THREAD

**Issue by RangerKevin2** [weihuoya/citra#342]
There are a lot of updates but we don't know what has been modified, so it would be cool if a citra built-in changelog was added.

**Comment by TheOG00** [comment]
@RangerKevin2 read the source code changes

**Comment by aecarvalho** [comment]
Absolutely, GitHub isn't Google Play Store, do read up on all the commits done since the last release.
Also, copy of issue #97
